src/elements/common/messages.js

```diff
@@ -132,6 +132,16 @@ const messages = defineMessages({
         description: 'Label for switching to grid view',
         defaultMessage: 'Switch to Grid View',
     },
+    gridViewIncreaseNumberOfColumns: {
+        id: 'be.gridViewIncreaseNumberOfColumns',
```

Comment:
Should we start namespacing these ids to group them together, like `be.gridView.increaseColumnCount`?